ci-operator/step-registry/workers-rhel7/aws-provision/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
ci-operator/step-registry/workers-rhel7/OWNERS file is empty....

Reply:
Fixed.